src/test/java/com/glovoapp/planningpoker/e2e/voting/AverageTest.java

```diff
@@ -0,0 +1,70 @@
+package com.glovoapp.planningpoker.e2e.voting;
+
+import com.glovoapp.planningpoker.e2e.WebDriverFactory;
+import com.glovoapp.planningpoker.e2e.pages.MainPage;
+import com.glovoapp.planningpoker.e2e.pages.SessionPage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.Optional;
+
+import static com.glovoapp.planningpoker.e2e.WebDriverFactory.getDriver;
+
+class AverageTest {
+
+    private final WebDriver driver = getDriver();
+    private final WebDriver driver2 = getDriver();
```

Comment:
Please don't use digits for variable names.

```suggestion
    private final WebDriver secondDriver = getDriver();
```